scripts/deploy.ts

```diff
@@ -311,6 +311,24 @@ class DeploymentExecutor {
     }
   }
 
+  async deployMockToken() {
+    //only deploy the mock for local environment using default deployer address
+    if (this.env == 'hardhat') {
```

Comment:
`if` is not really needed here, since the same check is done before calling `deployMockToken`.
Anyway, it does not hurt if you keep it as is (my approval will not depend on this).

Reply:
I modified the script to check network instead of env. I took the` if` out of the main function and left it in here.